perseo-utils/src/main/java/com/telefonica/iot/perseo/utils/DateTimeUtils.java

```diff
@@ -0,0 +1,81 @@
+package com.telefonica.iot.perseo.utils;
+
+import ca.rmen.sunrisesunset.SunriseSunset;
+import org.apache.log4j.pattern.DatePatternConverter;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
+import java.util.Calendar;
+
+public class DateTimeUtils {
```

Comment:
Are the methods included in this class properly covered by unit tests?

Reply:
Fixed in [f01e6a3](https://github.com/telefonicaid/perseo-core/pull/143/commits/f01e6a34ebc65a7ca3cb90e29f4bcadcc1a4e4a3)